### PR TITLE
Luca-AlexanderThomas.AutoHome-Autostart version 1.0.0

### PR DESCRIPTION
**Pull request by luca-alexander-thomas**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28071)

### CODE REVIEW

File: manifests/l/Luca-AlexanderThomas/AutoHome-Autostart/1.0.0/Luca-AlexanderThomas.AutoHome-Autostart.locale.de-DE.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Luca-AlexanderThomas.AutoHome-Autostart
+PackageVersion: 1.0.0
+PackageLocale: de-DE
+Publisher: Luca-Alexander Thomas
+PackageName: AutoHome-Autostart
```
Comment (ItzLevvie) [inline manifests/l/Luca-AlexanderThomas/AutoHome-Autostart/1.0.0/Luca-AlexanderThomas.AutoHome-Autostart.locale.de-DE.yaml:8]:
```suggestion
PackageName: AutoHome-Installer
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/l/Luca-AlexanderThomas/AutoHome-Autostart/1.0.0/Luca-AlexanderThomas.AutoHome-Autostart.locale.de-DE.yaml (hunk 1 above)
Comment (ItzLevvie) [inline manifests/l/Luca-AlexanderThomas/AutoHome-Autostart/1.0.0/Luca-AlexanderThomas.AutoHome-Autostart.locale.de-DE.yaml:8]:
```suggestion
PackageName: AutoHome
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/l/Luca-AlexanderThomas/AutoHome-Autostart/1.0.0/Luca-AlexanderThomas.AutoHome-Autostart.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: Luca-AlexanderThomas.AutoHome-Autostart
+PackageVersion: 1.0.0
+Installers:
+- InstallerLocale: de-DE
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/luca-alexander-thomas/AutoHome/releases/download/1.0/AutoHome-Installer-1.0.0_silent-install.msi
+  InstallerSha256: 7A2318E0856A5CCF2CCAAC7852EA4521617C6EC71FF4F5031A6A1AEDF0F417BE
+  ProductCode: '{DCF0C8FF-8909-4BBD-9147-5A5003E7464E}'
```
Comment (ItzLevvie) [inline manifests/l/Luca-AlexanderThomas/AutoHome-Autostart/1.0.0/Luca-AlexanderThomas.AutoHome-Autostart.installer.yaml:12]:
```suggestion
  ProductCode: '{DCF0C8FF-8909-4BBD-9147-5A5003E7464E}'
```
winget uninstall "{DCF0C8FF-8909-4BBD-9147-5A5003E7464E}" does not work.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/28071#pullrequestreview-758135449

File: manifests/l/Luca-AlexanderThomas/AutoHome-Autostart/1.0.0/Luca-AlexanderThomas.AutoHome-Autostart.installer.yaml (hunk 2 above)
Comment (luca-alexander-thomas) [reply manifests/l/Luca-AlexanderThomas/AutoHome-Autostart/1.0.0/Luca-AlexanderThomas.AutoHome-Autostart.installer.yaml:12]: ok are there no more mistakes in the Configuration?

**Review by luca-alexander-thomas — COMMENTED** [review]
another edit

**Comment by luca-alexander-thomas** [comment]
ok what have i to do next?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**